modules/statics/src/ofc.ts

```diff
@@ -554,3 +554,105 @@ export function tofcHederaToken(
     })
   );
 }
+
+/**
+ * Factory function for ofc arbethErc20 token instances.
+ *
+ * @param id uuid v4
+ * @param name unique identifier of the coin
+ * @param fullName Complete human-readable name of the coin
+ * @param network Network object for this coin
+ * @param decimalPlaces Number of decimal places this coin supports (divisibility exponent)
+ * @param asset Asset which this coin represents. This is the same for both mainnet and testnet variants of a coin.
+ * @param kind Differentiates coins which represent fiat assets from those which represent crypto assets
+ * @param prefix? Optional coin prefix. Defaults to empty string
+ * @param suffix? Optional coin suffix. Defaults to coin name.
+ * @param isToken? Whether or not this account coin is a token of another coin
+ * @param features? Features of this coin. Defaults to the DEFAULT_FEATURES defined in `OfcCoin`
+ * @param primaryKeyCurve The elliptic curve for this chain/token
+ */
+export function ofcArbethErc20(
```

Comment:
```suggestion
export function ofcArbethToken(
```

Reply:
Actually the onchain token is named as arbethErc20, so named it like this to maintain consistency. It might be confusing to name it as token for ofc alone.

Reply:
disregard. I see now that those tokens are called arbetherc20. Seems weird